webapp/src/main/java/com/box/l10n/mojito/service/scheduledjob/ScheduledJobTypeRepository.java

```diff
@@ -2,7 +2,9 @@
 
 import org.springframework.data.jpa.repository.JpaRepository;
 import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.rest.core.annotation.RepositoryRestResource;
 
+@RepositoryRestResource(exported = false)
```

Comment:
Just clarifying here, setting exported to false here ensures that we don't expose this Repository over HTTP endpoints, right?

Reply:
Correct